improve some grammar (#96)
v-community/v_by_example: examples/methods/methods.md

## Exercises

> 1. Create a method for the type `Person` that determines if a person has a legal age or not.

Reviewer: ```suggestion
1. Create a method for the type `Person` that determines whether a person is underage or not.
```

Author: thank you @vbrazo  👌
